### PR TITLE
Remove stackview deps

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

File: pages/settings/PageSettingsDisplayBrief.qml [hunk 1]
```diff
@@ -19,8 +19,10 @@ Page {
 	// Use this intermediate model that is built when the page loads, to avoid changing the model
 	// while the radio button group sub-page is shown, as that causes the group options to be rebuilt.
 	property var _gaugesModel
-	C.StackView.onActivating: {
-		_gaugesModel = Global.systemSettings.briefView.centralGauges.value || []
+	onIsCurrentPageChanged: {
```
Comment (jpetrell) [inline pages/settings/PageSettingsDisplayBrief.qml:22]: Can remove import QtQuick.Controls as C also.

File: pages/settings/PageSettingsWifi.qml [hunk 2]
```diff
@@ -19,7 +19,11 @@ Page {
 		settingsListView.model = Connman.getServiceList("wifi")
 	}
 
-	C.StackView.onActivated: _reload()
+	onIsCurrentPageChanged: {
```
Comment (jpetrell) [inline pages/settings/PageSettingsWifi.qml:22]: Can remove import QtQuick.Controls as C also.

**Comment by jpetrell** [comment]
\o/ Was just going to start looking into this, but you beat me to it (I don't mind :smile:). Fixes the issue beautifully. LGTM. :ok_hand:

File: pages/settings/PageSettingsDisplayBrief.qml (hunk 1 above)
Comment (blammit) [reply pages/settings/PageSettingsDisplayBrief.qml:21]: Done

File: pages/settings/PageSettingsWifi.qml (hunk 2 above)
Comment (blammit) [reply pages/settings/PageSettingsWifi.qml:21]: Done

**Pull request merged**